gcb.md

```diff
@@ -0,0 +1,87 @@
+# `kind` on Google Cloud Build
+
+## Example `cloudbuild.yml`
+
+The following cloud build config can be used by running
+
+```sh
+gcloud builds submit --config cloudbuild.yml --no-source
+```
+
+and will:
+
+- download current version of `kubectl`
+- compile `kind` from the tip of master
+- run `kind` to create a cluster
+- start a portforwarder to make the kube-apiserver accessible
+
+There are two things to be configured via `--substitutions`:
+- `KIND_CONFIG`: a [configuration for kind][kind-config] 
+- `KIND_TESTS`: whatever should be run after the cluster hast been created, `KUBECONFIG` is setup for you
+
+[kind-config]: https://kind.sigs.k8s.io/docs/user/quick-start/#configuring-your-kind-cluster
+
+```yaml
+steps:
+- name: 'gcr.io/cloud-builders/git'
+  args:
+  - clone
+  - https://github.com/kubernetes-sigs/kind
+  - src/kind
+- name: 'gcr.io/cloud-builders/docker'
+  entrypoint: bash
+  dir: bin
+  args:
+  - -c
+  - |
+    curl -L https://storage.googleapis.com/kubernetes-release/release/`curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt`/bin/linux/amd64/kubectl > kubectl
+    chmod 0750 kubectl
+- name: golang
```

Comment:
we should consider using the makefile to build (and get version info) or downloading a release binary

however, given the delay on this, I'd say we can think about that in a follow-up

Reply:
Which Makefile are you thinking about? sigs.k8s.io/kind/Makefile?

I can look into that.